### PR TITLE
fix GetNonce error

### PR DESCRIPTION
**Pull request by vanphandinh**
Error logs:
Screen Shot 2021-04-01 at 9 32 28 AM

### CODE REVIEW

**Comment by vanphandinh** [comment]
@jrwashburn can you check it?

**Comment by jrwashburn** [comment]
> @jrwashburn can you check it?

I will run a payout today to try to reproduce the error. Are you sending to a daemon endpoint or somewhere else?

**Comment by vanphandinh** [comment]
> > @jrwashburn can you check it?
> 
> I will run a payout today to try to reproduce the error. Are you sending to a daemon endpoint or somewhere else?

https://minagraph.com/graphql

You can test it directly.


```
    query GetNonce($publicKey: String) {
      account(publicKey: $publicKey) {
        inferredNonce
      }
    }
```

File: src/core/send-payments.ts [hunk 1]
```diff
@@ -60,7 +60,7 @@ async function sendSignedPayment (payment: signed<payment>) {
 
 export async function getNonce (publicKey: string) {
   const operationsDoc = `
-    query GetNonce($publicKey: String) {
+    query GetNonce($publicKey: PublicKey!) {
```
Comment (jrwashburn) [inline src/core/send-payments.ts:63]: Confirmed change works against mina daemon. Looks good.

**Review by jrwashburn — APPROVED** [review]

**Pull request merged**